frontend/src/components/ChangeProfileForm/ChangeProfileForm.js

```diff
@@ -0,0 +1,201 @@
+import React, { useState } from 'react';
+import {
+  Button,
+  CardContent,
+  IconButton,
+  InputAdornment,
+  Grid,
+  Paper,
+  TextField,
+  Typography,
+  Avatar,
+} from '@material-ui/core';
+import { visibilityIcon, visibilityOffIcon } from '../../assets';
+import { EMAIL_REGEX, PASSWORD_REGEX, SUNFLOWER } from '../../constants';
+
+export default function ChangeProfileForm({
+  firstName,
+  lastName,
+  mail,
+  onSaveChanges,
+}) {
+  const [email, setEmail] = useState(mail);
+  const [password, setPassword] = useState('');
+  const [confirmPassword, setConfirmPassword] = useState('');
+  const [showPassword, setShowPassword] = useState(false);
+  const [showConfirmPassword, setShowConfirmPassword] = useState(false);
+  const [showError, setShowError] = useState({
+    errorEmail: false,
+    errorPassword: false,
+    errorConfirmPassword: false,
+  });
+
+  const handleSaveChanges = (event) => {
+    event.preventDefault();
+    if (!EMAIL_REGEX.test(email)) {
+      setShowError({ ...showError, errorEmail: true });
+      return;
+    }
+    if (!PASSWORD_REGEX.test(password)) {
+      setShowError({ ...showError, errorPassword: true });
+      return;
+    }
+    if (confirmPassword !== password) {
+      setShowError({ ...showError, errorConfirmPassword: true });
+      return;
+    }
+    onSaveChanges(email, password);
+  };
+
+  const handleEmailChange = (event) => {
+    setEmail(event.target.value);
+    setShowError({ ...showError, errorEmail: false });
+  };
+  const handlePasswordChange = (event) => {
+    setPassword(event.target.value);
+    setShowError({ ...showError, errorPassword: false });
+  };
+
+  const handleConfirmPasswordChange = (event) => {
+    setConfirmPassword(event.target.value);
+    setShowError({ ...showError, errorConfirmPassword: false });
+  };
+
+  const handleClickShowPassword = () => setShowPassword(!showPassword);
+  const handleClickShowConfirmPassword = () =>
+    setShowConfirmPassword(!showConfirmPassword);
+
+  return (
+    <Paper elevation={3}>
+      <CardContent>
+        <Grid item xs={12}>
+          <Grid container spacing={3} justify="flex-start" alignItems="center">
+            <Grid item>
+              <Avatar style={{ backgroundColor: SUNFLOWER }}>
+                {firstName[0] + lastName[0]}
+              </Avatar>
+            </Grid>
+            <Grid item>
+              <Typography variant="h6">{`${firstName} ${lastName}`}</Typography>
+            </Grid>
+          </Grid>
+        </Grid>
```

Comment:
Add more spacing between this Grid item and form below. Now the top elements look squashed with email text field

Reply:
done